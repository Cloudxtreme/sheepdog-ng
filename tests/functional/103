#!/bin/bash

# Test node events with vnodes unchanged

. ./common

for i in `seq 0 2`; do
    _start_sheep $i
done

_wait_for_sheep 3

_cluster_format -c 2

$DOG vdi create test 100M
_random | $DOG vdi write test

for i in `seq 3 6`; do
    _start_sheep $i -g
done

_wait_for_sheep 7

$DOG vdi check test

_kill_sheep 3
_wait_for_sheep 6

grep recovery $STORE/0/sheep.log

_kill_sheep 1
_kill_sheep 2
_kill_sheep 4
_kill_sheep 6
_wait_for_sheep 2

_start_sheep 2
_wait_for_sheep 3

if [ "`grep identical $STORE/0/sheep.log`" ]; then
    echo "Yes, we did skip the identical epoch"
fi

$DOG vdi check test
$DOG node list

