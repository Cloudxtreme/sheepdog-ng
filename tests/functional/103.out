QA output created by 103
using backend plain store
finish check&repair test
Yes, we did skip the identical epoch
finish check&repair test
  Id   Host:Port         V-Nodes       Zone
   0   127.0.0.1:7000      	128          0
   1   127.0.0.1:7002      	128          2
   2   127.0.0.1:7005      	 0          5
